web/libs/ui/src/lib/MessageToast/MessageToast.tsx

```diff
@@ -1,24 +1,30 @@
-import { Toast, ToastAction, type ToastProps } from "../../components/Toast/Toast";
+import { Toast, ToastAction, ToastType, type ToastProps } from "../Toast/Toast";
 import { LsCross } from "../../assets/icons";
 import type { FC } from "react";
 import { Block } from "../../utils/bem";
```

Comment:
BEM cannot be used in the ui library.

Reply:
I had reservations on this tbh, but this ticket has already exploded in scope so i opted to go down this route. you are right though i'll get rid of BEM in Toast and MessageToast in favor of scss modules